presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,60 @@
+# Release 348 (XX Dec 2020)
+
+## General Changes
+
+* Add support for `DISTINCT` clause in aggregations within correlated subqueries. ({issue}`5904`)
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`6126`)
+* Reduce memory pressure and improve performance of queries that contain join. ({issue}`6176`)
+* Fix ``EXPLAIN ANALYZE`` for certain queries that contain broadcast join. ({issue}`6115`)
+* Fix planning failures for queries that contain filtered aggregations and outer joins. ({issue}`6141`)
```

Comment:
What are "filtered aggregations"?

Reply:
https://prestosql.io/docs/current/functions/aggregate.html#filtering-during-aggregation

Reply:
Thanks. Maybe we should quote the syntax in the release notes:
```
Fix planning failures for queries that contain outer joins and aggregations using `FILTER (WHERE <condition>)` syntax.
```